tests/emb6/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = emb6
+include ../Makefile.tests_common
```

Comment:
Here it was also excluded because of the `BOARD` setting. #6121 might allow to run `emb6` on native.